### PR TITLE
#232 Fix favicon rendering logic and add Links

### PR DESCRIPTION
**Pull request by asynchroza**

### CODE REVIEW

**Review by SuelaCark — CHANGES_REQUESTED** [review]
For some reason, every time you go from HackAUBG to the landing page through clicking the Hub Logo, the favicon still stays green unless the page is either refreshed or you click on the jobs section.

**Comment by asynchroza** [comment]
@SuelaCark @Aleks1908 up for verification

File: packages/web/src/components/spa/AboutSection/AboutSection.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import './about_section.css';
 
 export const AboutSection = () => {
     return (
-        <div className="about-container" id="AboutSection">
+        <div className="about-container" id="about">
             <div className="about-image">
```
Comment (SuelaCark) [inline packages/web/src/components/spa/AboutSection/AboutSection.jsx:7]: To what I noticed, the about section anchor in the landing page is mistakenly connected to the one from the hackaubg instead. Hence, when you click the About anchor in Hackaubg nothing happens, but when you got then to the landing page, it loads immediately to the about us section.

File: packages/web/src/components/spa/MainPage.jsx [hunk 2]
```diff
@@ -12,9 +12,9 @@ import { checkHashAndScroll } from '../../Global';
 
 const LandingHome = () => {
     const anchorList = [
-        new Anchor('About', '#AboutSection'),
+        new Anchor('About', '#about'),
         new Anchor('Team', '#team'),
```
Comment (SuelaCark) [inline packages/web/src/components/spa/MainPage.jsx:16]: As mentioned above, please make sure the About section anchor for landing page is connected correctly.

**Review by SuelaCark — CHANGES_REQUESTED** [review]
Please check the comments. Everything else about the favicon works perfectly.

**Comment by asynchroza** [comment]
@SuelaCark please verify again. I couldn't come up with a clean solution on how to load at top level and I don't have the time to do so. The implemented scrolling looks nice to me, if it's alright with you, feel free to approve and merge

**Review by SuelaCark — APPROVED** [review]

**Pull request merged**